cypress/integration/our-story-spec.js

```diff
@@ -1,19 +1,42 @@
 describe('Our Story page user flow', () => {
   beforeEach(() => {
     cy.checkPageA11y('/our-story')
+    cy.get('.en-espanol-button').select('English')
   });
 
-  it('Should see all four developers\' names, photo, title, and reason for working on the app', () => {
-    cy.fixture('devs.json').as('devs').then((devs) => {
-      const checkAllDevs = devs.map(dev => {
-      cy.get('.dev-name').should('contain', dev.name)
-        .get(`.${dev.name.split(' ')[0]}`).should('be.visible')
-        .get(`.${dev.name.split(' ')[0]}`).should('have.attr', 'alt')
-        .should('contain', dev.name)
-        .get('.dev-title').should('contain', dev.title)
-        .get('.dev-story').should('contain', dev.story)
+  it('Should see all four devs\' names, photo, title, & reason for working on the app, in English & Spanish, ', () => {
+    cy.fixture('english.json').as('english').then((english) => {
+      cy.fixture('devs.json').as('devData').then((devData) => {
+        const checkAllDevData = devData.map(developer => {
+          const checkAllDevTranslations = english.devs.map(dev => {
+            cy.get('.dev-name').should('contain', developer.name)
+              .get(`.${developer.name.split(' ')[0]}`).should('be.visible')
+              .get(`.${developer.name.split(' ')[0]}`).should('have.attr', 'alt')
+              .should('contain', developer.name)
+              .get('.dev-title').should('contain', dev.title)
+              .get('.dev-story').should('contain', dev.story)
+          })
+        })
+      })
+    })
+
+    cy.fixture('spanish.json').as('spanish').then((spanish) => {
+      cy.get('.en-espanol-button').select('Español')
+      cy.fixture('devs.json').as('devData').then((devData) => {
+        const checkAllDevData = devData.map(developer => {
+          const checkAllDevTranslations = spanish.devs.map(dev => {
+            cy.get('.dev-name').should('contain', developer.name)
+              .get(`.${developer.name.split(' ')[0]}`).should('be.visible')
+              .get(`.${developer.name.split(' ')[0]}`).should('have.attr', 'alt')
+              .should('contain', developer.name)
+              .get('.dev-title').should('contain', dev.title)
+              .get('.dev-story').should('contain', dev.story)
+          })
+        })
```

Comment:
These were tricky because of the need to cross-reference stubbed data from two different sources.  The original developer data that was written for iteration one, and then the translated data that was only translated for the dev's title and dev's story.  Equating the name, linkedin, github, all of that data came from devs.json.